tap_google_ads/streams.py

```diff
@@ -285,6 +286,24 @@ def __init__(self, fields, google_ads_resource_names, resource_schema, primary_k
 
         self.build_stream_metadata()
 
+    
+    def get_custom_fields(self, fields_dict: dict, stream_name: str)  -> Optional[list]:
+        """Getting custom fields of the stream
+
+        Args:
+            fields (Optional[dics]): a dict with custon fields if defined
```

Comment:
minor thing, but here we say 'field' is an Optional[dict] and in function type hint we say it's 'dict', can we unify it?